### DISCUSSION TITLE
Is it possible to use Threejs materials?

### DISCUSSION THREAD

**Discussion by 5p4m3r** [google/model-viewer#4586] [Q&A]
Hello,

I would like to use the Threejs Matcap material? 
How could I implement it?

thx in advance.

**Comment by elalish** [comment]
You'd need to fork us for that. Go for it!

**Reply by 5p4m3r** [reply]
I was expecting something along those lines. Unfortunately this is outside of my abilities. I was hoping for a more accessible solution. Is there an unofficial example on how to access the underlying threejs scene floating around?

**Reply by elalish** [reply]
There's this: https://modelviewer.dev/docs/faq.html#entrydocs-general-questions-three
